agent/agent.go

```diff
@@ -112,13 +117,14 @@ func (a *Agent) Backup(bcp pbm.BackupCmd) {
 	err = backup.New(a.pbm, a.node).Run(bcp)
 	if err != nil {
 		log.Println("[ERROR] backup:", err)
+	} else {
+		log.Printf("Backup %s finished", bcp.Name)

```

Comment:
Shouldn't the log message for the user be 'backup xxx started' rather than 'backup xxx finished'?

Reply:
"Backup started" message is few lines above. At this, point backup is finished (if `Run(bcp)` returned no error).

Reply:
Ah, I think I see. This would only be in the very untypical case the whole backup completes in < StaleFrameSec.